app.js

```diff
@@ -2,20 +2,19 @@ const express = require("express");
 const app = express();
 const { allTopics } = require("./controllers/topicController");
 const {
-  pullArticle,
+  allArticles,
```

Comment:
remember the fundamentals! Functions should describe an action. `allArticles` is a noun, not an action.